src/pipeline/src/manager/table.rs

```diff
@@ -325,24 +302,100 @@ impl PipelineTable {
 
         {
             self.pipelines.insert(
-                Self::generate_pipeline_cache_key(schema, name, None),
+                generate_pipeline_cache_key(schema, name, None),
                 compiled_pipeline.clone(),
             );
             self.pipelines.insert(
-                Self::generate_pipeline_cache_key(schema, name, Some(TimestampNanosecond(version))),
+                generate_pipeline_cache_key(schema, name, Some(TimestampNanosecond(version))),
                 compiled_pipeline.clone(),
             );
```

Comment:
Corner case:

```
create v0                      | create v1
put my_schema/my_table/None    |
                               | put my_schema/my_table/None
                               | put my_schema/my_table/Latest
put my_schema/my_table/latest  |
```

Reply:
Looks fine. Because there is a 10-second TTL, the eventual consistency in this API is enough.